backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -184,7 +184,7 @@ const getOptInCode = async (loginUserId: number): Promise<LoginEmailOptIn> => {
   // Check for `CONFIG.EMAIL_CODE_VALID_TIME` minute delay
   if (optInCode) {
     if (isOptInCodeValid(optInCode)) {
-      throw new Error(`email already sent less than $(CONFIG.EMAIL_CODE_VALID_TIME} minutes ago`)
+      throw new Error(`email already sent less than $(printEmailCodeValidTime()} minutes ago`)
```

Comment:
this must be a different parameter, which still is 10mins, since if the user loses his code he can not request another for 24h with this code